### ISSUE TITLE
How much epoch do it need to run？

### ISSUE THREAD

**Issue by cangwang** [ptran1203/pytorch-animeGAN#13]
I have run 100 epoch for shikai dataset ，and adv loss is more and more larger。and the result is not real like shikai style and do it wrong  to get result？

**Comment by ptran1203** [comment]
You can try to tune the hyperparameters

```
                --wadvd 10.0\               # Aversarial loss weight for D
                --wadvg 10.0\               # Aversarial loss weight for G
                --wcon 1.5\                 # Content loss weight
                --wgra 3.0\                 # Gram loss weight
                --wcol 30.0\                # Color loss weight
```

And consider stop training if the loss start increase

**Comment by cangwang** [comment]
You have use how many epoch to change？ Use tune hyperparameters is another function to change ，and we can‘’t feel the source image and fake images direct。
